samza-core/src/test/scala/org/apache/samza/job/TestShellCommandBuilder.scala

```diff
@@ -44,7 +44,7 @@ class TestShellCommandBuilder {
 
   // if cmdPath is specified, the full path to the command should be adjusted
   @Test
-  def testCommandWithFwkPath {
+  def testBuildCommandWithCommandPath {
```

Comment:
Did you mean 'testBuildCommandWithPackagePath'?

Reply:
I intended to use "command path". There is a method `ShellCommandBuilder.setCommandPath`, so I made the test method consistent with that.